pkgs/applications/emulators/bsnes/higan/default.nix

```diff
@@ -52,12 +52,9 @@ stdenv.mkDerivation (finalAttrs: {
     openal
     udev
   ]
-  ++ lib.optionals stdenv.hostPlatform.isDarwin (with darwin.apple_sdk.frameworks; [
-    Carbon
-    Cocoa
-    OpenAL
-    OpenGL
-  ]);
+  ++ lib.optionals stdenv.hostPlatform.isDarwin [
+    apple-sdk
```

Comment:
This does nothing. `apple-sdk` is already present in the standard environment. It’s where the libc comes from.

Reply:
TIL
(On the other hand I believe SDK 11 would be more suitable.)